docs/hooks.md

```diff
@@ -91,10 +93,18 @@ hooks.beforeEach(function (transaction) {
   hooks.log('beforeEach');
 });
 
+hooks.beforeEachValidation(function (transaction) {
```

Comment:
I wonder - didn't we wanted to keep the `order` of hooks as it actually should be in this one-long-example?

So please move it after `before`.